### DISCUSSION TITLE
need a local simulator

### DISCUSSION THREAD

**Discussion by squirrelsc** [microsoft/nni#3767] [Ideas]
Sometime, if there is any error in trial, tuner or accessor code, it's hard to troubleshooting. If there is a local simulator for NNI, that would be easier.

For example, user can run it like
nnictl simulate --config mnist.yaml

It uses mockup nni manager, dispatcher, and collect all log, output to console or single log file. User can get any runtime error in trial, tuner and accessor code immediately.

**Comment by QuanluZhang** [comment]
@squirrelsc  this can be achieved using standalone mode?

**Comment by squirrelsc** [comment]
Does this command support? Can standalone mode input with yaml file, and output all log in console or single log file?
